src/test/java/plugins/MatrixAuthPluginTest.java

```diff
@@ -48,14 +49,23 @@ public void helloWorld() throws Exception {
         jenkins.login().doLogin("alice");
 
         FreeStyleJob j = jenkins.jobs.create();
+
         j.save();
 
         // if we login as Bob, he shouldn't see the job
         jenkins.login().doLogin("bob");
+
+        //wait for main panel to appear to make sure page is rendered
+        waitFor(by.id("main-panel"));
+
         assertNull(getElement(by.href("job/"+j.name+"/")));
 
         // contorl assertion: alice shoudl see the link
         jenkins.login().doLogin("alice");
+
+        //wait for main panel to appear to make sure page is rendered
+        waitFor(by.id("main-panel"));
```

Comment:
The layout could change but not the ID of that element. And if it changes [this is not going to be the only test failing](https://github.com/jenkinsci/acceptance-test-harness/search?utf8=%E2%9C%93&q=main-panel).